src/ign-3to2

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/python3
+import os
+import sys
+import json
+import shutil
+import argparse
+
+# Parse args and dispatch
+parser = argparse.ArgumentParser()
+parser.add_argument("path", help="Ignition file")
+args = parser.parse_args()
+
+if args.path == '-':
+    f = sys.stdin
+else:
+    f = open(args.path)
+ign = json.load(f)
+v = ign['ignition']['version']
+if v != '3.0.0':
```

Comment:
nit: It would be nice to return out if the version is `2.x.x`. Not a requirement though!

Reply:
Done